openc3/lib/openc3/script/storage.rb

```diff
@@ -104,11 +104,12 @@ def get_target_file(path, original: false, scope: $openc3_scope)
           if part == "targets_modified" and ENV['OPENC3_LOCAL_MODE']
             local_file = OpenC3::LocalMode.open_local_file(path, scope: scope)
             if local_file
+              OpenC3::Logger.info "Reading local #{scope}/#{path}"
```

Comment:
Do we want to log reading files?

Reply:
Yeah we're already doing it in the nominal case so we should in the local mode case